portfolio/src/main/java/com/google/sps/servlets/MessageFormServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/message-form-handler")
```

Comment:
Is this still used anywhere? It looks like the index.html is hitting the /new-message endpoint instead

Let's remove this if it's not used anymore